<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageThreshProcess</class>
 <widget class="QWidget" name="ImageThreshProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>691</width>
    <height>598</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>691</width>
    <height>598</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageThreshProcess</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ImagePseudoColor.ico</normaloff>ImagePseudoColor.ico</iconset>
  </property>
  <widget class="QwtPlot" name="qwtPlot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>311</height>
    </rect>
   </property>
   <property name="canvasBackground">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </brush>
   </property>
   <property name="autoReplot">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_OrignianlImage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string>OrignianlImage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Preoss">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveQuit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save&amp;&amp;Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set To</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Higher than</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_SetToValue">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ThresholdValue">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
